core/src/main/java/nl/nn/adapterframework/doc/objects/IbisBean.java

```diff
@@ -0,0 +1,28 @@
+package nl.nn.adapterframework.doc.objects;
+
+public class IbisBean implements Comparable<IbisBean> {
```

Comment:
Wat representeert deze class?

Reply:
Ik heb dit niet geschreven, enkel verplaatst om de IbisDocPipe.class wat overzichtelijker te maken zodat er niet meerder classes zijn in een file.

Reply:
Het representeert tevens een Class Object